homeassistant/components/tplink/__init__.py

```diff
@@ -53,10 +66,69 @@ def async_trigger_discovery(
         )
 
 
+def _decrypt(encrypted: str) -> str:
```

Comment:
Not worth separate encryption here as such information is usually stored unencrypted on the disk anyway in homeassistant.